src/ModelOrderReduction.jl

```diff
@@ -7,6 +7,10 @@ using LinearAlgebra
 using TSVD
 using RandomizedLinAlg
 
+import TSVD.tsvd
+import RandomizedLinAlg.rsvd
+import LinearAlgebra.svd
```

Comment:
This makes it piracy because it's then adding a function on Base types. It would be safer to just shadow or define a `_tsvd` etc. inside of the package for the Vector{Vector} form.

Reply:
good point. Didn't think about that whatsoever. Should be fixed now.